test/com/google/enterprise/adaptor/TestHelper.java

```diff
@@ -87,17 +89,6 @@ public void setAuthzAuthority(AuthzAuthority authzAuthority) {}
     return getDocIds(adaptor, Collections.<String, String>emptyMap());
   }
 
-  public static byte[] getDocContent(Adaptor adaptor, DocId docId)
```

Comment:
TestHelper is public and so is this method, so they are both available to connectors. I don't think you should remove this method.